src/vue/components/setting.vue

```diff
@@ -58,7 +58,7 @@ export default {
       required: true
     },
     selectedOverride: {
-      type: Object,
+      type: String,
```

Comment:
As we pass `:selected-override="$store.state.app.currentLanguageName"`
I think it is a right fix!
Looks good!